work-with-collection-array-list

ap2144839 opened the pull request:
- creating class com.peterson.collection.WorkWithArrayList
- creating methods for class WorkWithArrayList:
   fillList(int size), deleteElements(WorkWithArrayList list, int index)
- modification of class com.peterson.collection.Util:
   time measuring of creation ArrayList collection and deleting one elements from it
https://trello.com/c/F9PUxJtS/2-work-with-collection-arraylist

ap2144839 committed:
work-with-collection-array-list
- creating class com.peterson.collection.WorkWithArrayList
- creating methods for class WorkWithArrayList:
   fillList(int size), deleteElements(WorkWithArrayList list, int index)
- modification of class com.peterson.collection.Util:
   time measuring of creation ArrayList collection and deleting one elements from it
https://trello.com/c/F9PUxJtS/2-work-with-collection-arraylist

Alexis220382 commented on src/com/peterson/collection/WorkWithArrayList.java:
```diff
@@ -0,0 +1,24 @@
+    public ArrayList list;
+
+    public ArrayList fillList(int size){
+        list = new ArrayList<>();
```
Если ты хочешь указать на поле класса, то лучше использовать ключевое слово this

ap2144839 replied:
list = new ArrayList<>();

ap2144839 replied:
`this.list = new ArrayList<Integer>();`

Alexis220382 replied:
Исправления делай не в текстовых комментах, а в коде и обновляй pull request новым коммитом и пушем из этой же ветки

Alexis220382 commented on src/com/peterson/collection/WorkWithArrayList.java:
```diff
@@ -0,0 +1,24 @@
+    public ArrayList fillList(int size){
+        list = new ArrayList<>();
+        for (int i = 0; i < size; i++){
+            list.add(1);
```
Тут ты полный лист единиц решил сделать?

ap2144839 replied:
`list.add(1000);`

Alexis220382 commented on src/com/peterson/collection/WorkWithArrayList.java:
```diff
@@ -0,0 +1,24 @@
+        return list;
+    }
+
+    public void deleteElementList (WorkWithArrayList list, int index) {
```
Непонятно зачем в сигнатуре метода класса находится сам класс???
Может ты туда List хотел вставить?

ap2144839 replied:
`public void deleteElementList (List list, int index) {`

Alexis220382 commented on src/com/peterson/collection/WorkWithArrayList.java:
```diff
@@ -0,0 +1,24 @@
+ */
+public class WorkWithArrayList extends WorkWithList {
+
+    public ArrayList list;
```
Насколько я помню, в задаче был такой пример поля класса `ArrayList<Integer>`

ap2144839 replied:
`public ArrayList<Integer> list;`

Alexis220382 replied:
in WorkWithArrayList class create field with name 'list' and type of `ArrayList<Integer>`

ap2144839 committed:
work-with-collection-array-list
- changing class com.peterson.collection.WorkWithArrayList
- changing class com.peterson.collection.Util
https://trello.com/c/F9PUxJtS/2-work-with-collection-arraylist
